Formula/opentsdb.rb

```diff
@@ -0,0 +1,139 @@
+class HbaseLZORequirement < Requirement
+  fatal true
+
+  satisfy(:build_env => false) { !Tab.for_name("hbase").without?("lzo") }
+
+  def message; <<-EOS.undent
+  hbase must not have disabled lzo compression to use it in opentsdb:
+    brew install hbase
+    not
+    brew install hbase --without-lzo
+  EOS
+  end
+end
+
+class Opentsdb < Formula
+  desc "Scalable, distributed Time Series Database."
+  homepage "http://opentsdb.net/"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.2.0/opentsdb-2.2.0.tar.gz"
+  sha256 "5689d4d83ee21f1ce5892d064d6738bfa9fdef99f106f45d5c38eefb9476dfb5"
+
+  option "without-lzo", "Don't use LZO Compression"
+
+  if build.without? "lzo"
+    depends_on "hbase"
+  else
+    depends_on "hbase"
+    depends_on "lzo"
+    depends_on HbaseLZORequirement
+  end
+  depends_on :java => "1.6+"
+  depends_on "gnuplot" => :optional
+
+  def install
+    # submitted to upstream https://github.com/OpenTSDB/opentsdb/pull/711
+    # mkdir_p is called from in a subdir of build so needs an extra ../ and there is no rule to create $(classes) and
+    # everything builds without specifying them as dependencies of the jar.
+    inreplace "Makefile.in" do |s|
+      s.sub!(/(\$\(jar\): manifest \.javac-stamp) \$\(classes\)/, '\1')
+      s.sub!(/(echo " \$\(mkdir_p\) '\$\$dstdir'"; )/, '\1../')
+    end
+
+    mkdir "build" do
+      system "../configure",
+             "--disable-silent-rules",
+             "--prefix=#{prefix}",
+             "--mandir=#{man}",
+             "--sysconfdir=#{etc}",
+             "--localstatedir=#{var}/opentsdb"
+      system "make"
+      system "make", "install-exec-am"
+      system "make", "install-data-am"
+    end
+
+    env = Language::Java.java_home_env.merge(:HBASE_HOME => Formula["hbase"].opt_libexec, :COMPRESSION => (build.with?("lzo") ? "LZO" : "NONE"))
+    (bin/"create_table.sh").write_env_script opt_pkgshare/"tools/create_table.sh", env
+  end
+
+  def post_install
+    (var/"cache/opentsdb").mkpath
+    inreplace opt_pkgshare/"etc/opentsdb/opentsdb.conf", "usr/share", "usr/local/share"
```

Comment:
What is this replacing? If that's replacing `"/usr/share"` with `"/usr/local/share"`, then the latter should probably be `"#{HOMEBREW_PREFIX}/share"` so that things continue to work for Homebrew users who install outside of `/usr/local`.